src/GPL.txt

```diff
@@ -1,340 +0,0 @@
-		    GNU GENERAL PUBLIC LICENSE
```

Comment:
Idk how it really goes, but original `kops/src/README.txt` says that

> You should have received a copy of the GNU General Public License
> along with this program; if not, write to the Free Software
> Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA

Should we keep this? Or update it to GPL3?

Reply:
We already have `LICENSE` in the root, this was basically a duplicate (though yes, this is GPL2 and the LICENSE in the root is GPL3).

Not sure which the original author wants.

Reply:
Oh yea, LICENSE is the whole thing. In original readme it is said that we can use newer version:

> This program is free software; you can redistribute it and/or modify
> it under the terms of the GNU General Public License as published by
> the Free Software Foundation; either version 2 of the License, or
> (at your option) any later version.

so we good.